### ISSUE TITLE
Internal Trade Routes giving double yields

### ISSUE THREAD

**Issue by RhysDeAnno** [LoneGazebo/Community-Patch-DLL#6869]
_1. Mod version (i.e Date - 4/17):_
6/19-2 with 7/29 DLL

_2. Mod list (if using Vox Populi only, leave blank):_
No other mods

_3. Error description:_ 
Internal Trade Routes appear to be giving double yields. On the trade pages the yields are normal, but in the cities the yields show and appear to function as double. Seems to work with land and sea, and also food and production. There's also some weird rounding going on there which may help to locate the issue. There are some screenshots from the save to show the quick idea of what's going on.

_4. Steps to reproduce (optional):_
Load the save and inspect the situation. There is a fresh caravan ready to fire too if you want to experiment more with the effect.

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

<img width="151" alt="Trade1" src="https://user-images.githubusercontent.com/47321731/89550173-4c85d080-d7d7-11ea-87c5-b4c47d228f33.png">

<img width="409" alt="Trade2" src="https://user-images.githubusercontent.com/47321731/89550195-51e31b00-d7d7-11ea-99d2-889ef578b57c.png">

<img width="319" alt="Trade3" src="https://user-images.githubusercontent.com/47321731/89550213-56a7cf00-d7d7-11ea-8c48-c76d8660e5e6.png">

<img width="332" alt="Trade4" src="https://user-images.githubusercontent.com/47321731/89550231-5d364680-d7d7-11ea-9f77-5ce23d640368.png">

[Trader Atilla.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/5035869/Trader.Atilla.zip)

**Comment by RhysDeAnno** [comment]
I'm not sure what you mean by "cannot reproduce" in this context. When you load the save, are the trade routes not producing the extra yields? I tried a reinstall of VP and re-copied the new DLL and the problem persisted, it also appeared when I loaded an older save, and even when I loaded a save of a previous game from this patch (I'm not sure if the problem was there before and I just didn't notice it.)

What sort of extra information can I provide? I have logs enabled but I don't know where this kind of thing would show up, since it seems like a static effect problem and not a bad triggered thing.
